src/diffusers/loaders.py

```diff
@@ -779,9 +838,20 @@ class LoraLoaderMixin:
     unet_name = UNET_NAME
 
     def load_lora_weights(self, pretrained_model_name_or_path_or_dict: Union[str, Dict[str, torch.Tensor]], **kwargs):
```

Comment:
This is a user-facing method, and I would prefer to have docstrings for more than the methods that are less user-facing.